spec/decorators/member_decorator_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe MemberDecorator do
+end
```

Comment:
You could test here your decorators methods, in the case you decide to refactor the two similar methods

Reply:
done